Tasks/AzureFileCopy/task.loc.json

```diff
@@ -11,9 +11,9 @@
   ],
   "author": "Microsoft Corporation",
   "version": {
-    "Major": 1,
+    "Major": 2,
```

Comment:
If this is a new major version it needs to start in preview along with a description about why the user should jump to your new preview (there's a whole experience around this)

Reply:
Added preview label and release notes